### DISCUSSION TITLE
zoning Madel in Home Assistant

### DISCUSSION THREAD

**Discussion by maurogrioni** [indomus/forum#3257] [HUB: Home Assistant]
ciao a tutti .
vorrei integrare il mio impianto di condizionamento controllato da Madel Zoning in Modbus
per poterlo controllare in wifi ho collegato un Elfin-EW1X.
ho fatto vari tentativi ma riesco a stabilizzare i sensori .
le temperature per esempio si vedono per un pò. poi spariscono, cosi come il controllo di accensione e spegnimento.
se qualcuno può aiutarmi
grazie

allego codici modbus e listato dei sensori
[Elfin-EW1X_User ManualV1.3(20200415).pdf](https://github.com/user-attachments/files/17445859/Elfin-EW1X_User.ManualV1.3.20200415.pdf)
[MANUALE TECNICO DI CONNESSIONE DEL BUS EXTERNO DELLA CENTRALE ZITY.pdf](https://github.com/user-attachments/files/17445860/MANUALE.TECNICO.DI.CONNESSIONE.DEL.BUS.EXTERNO.DELLA.CENTRALE.ZITY.pdf)
[Zoning Modbus BMS 19_01 (EN).pdf](https://github.com/user-attachments/files/17445861/Zoning.Modbus.BMS.19_01.EN.pdf)

 
[modbus-yaml.txt](https://github.com/user-attachments/files/17445923/modbus-yaml.txt)

**Comment by giuliomorsone** [comment]
Ciao, sei riuscito ad integrarlo? Anche io ho lo stesso sistema.

Grazie

Giulio

**Comment by maurogrioni** [comment]
No non sono riuscitoDiciamo che ci ho rinunciatoIl 14 ott 2025 22:32, giuliomorsone ***@***.***> ha scritto:
Ciao, sei riuscito ad integrarlo? Anche io ho lo stesso sistema.
Grazie
Giulio
